spec/components/candidate_interface/contact_details_review_component_spec.rb

```diff
@@ -15,12 +15,16 @@
 
   context 'when contact details are editable' do
     it 'renders component with correct values for a phone number' do
-      result = render_inline(described_class.new(application_form: application_form))
-
-      expect(result.css('.govuk-summary-list__key').text).to include(t('application_form.contact_details.phone_number.label'))
-      expect(result.css('.govuk-summary-list__value').text).to include('07700 900 982')
-      expect(result.css('.govuk-summary-list__actions a')[0].attr('href')).to include(Rails.application.routes.url_helpers.candidate_interface_edit_phone_number_path)
-      expect(result.css('.govuk-summary-list__actions').text).to include("Change #{t('application_form.contact_details.phone_number.change_action')}")
+      render_inline(described_class.new(application_form: application_form))
+
+      expect(rendered_component).to summarise(
```

Comment:
😍  I like the semantics! Should it take an array, as in most cases (I think?) we'll be asserting on multiple rows?

Reply:
Possibly?  Separate `expect` statements might be more readable? Not sure.

```ruby
expect(rendered_component).to summarise(
  key: 'Phone number',
   value: '07700 900 982',
   action: {
    text: 'Change phone number',
    href: Rails.application.routes.url_helpers.candidate_interface_edit_phone_number_path,
  }
)
  
expect(rendered_component).to summarise(
  key: 'Email address',
  value: 'test@example.com',
  action: {
    text: 'Change email address',
    href: Rails.application.routes.url_helpers.candidate_interface_edit_email_path,
  }
)
```

vs

```ruby
expect(rendered_component).to summarise(
  [
    {
      key: 'Phone number',
       value: '07700 900 982',
       action: {
        text: 'Change phone number',
        href: Rails.application.routes.url_helpers.candidate_interface_edit_phone_number_path,
      }    
    },
    {
      key: 'Email address',
      value: 'test@example.com',
      action: {
        text: 'Change email address',
        href: Rails.application.routes.url_helpers.candidate_interface_edit_email_path,
      }
    }
  ]
)
```

🤷

Reply:
I like the individual assertions from a readability point of view. Also makes it easy to work out what is wrong when an assertion fails.